sympy/tensor/array/ndim_array.py

```diff
@@ -586,6 +587,12 @@ def _check_index_for_getitem(self, index):
 
         return index
 
+    def _eval_evalf(self, n=15, subs=None, maxn=100, chop=False, strict=False, quad=None, verbose=False):
+        """Apply evalf() to each element of self."""
+        options = {'subs':subs, 'maxn':maxn, 'chop':chop, 'strict':strict,
+                'quad':quad, 'verbose':verbose}
+        return self.applyfunc(lambda i: i.evalf(n, **options))
```

Comment:
In this situation it would be better for the method to take `**options` and just pass them on rather than listing out all of the arguments explicitly.

Reply:
@oscarbenjamin I believe the new commit addresses this. Also, since the code originally came from the matrix class, let me know if I should open a parallel pull request to simplify that part of the code the same way.

Reply:
That could also be fixed. Actually looking at `git grep _eval_evalf` it does not look like `_eval_evalf` should take any arguments apart from `prec`.